doc/api/v0.yaml

```diff
@@ -230,6 +236,10 @@ components:
           items:
             $ref: "#/components/schemas/VotePlan"
           description: Vote plans registered for voting in this fund campaign.
+        groups:
```

Comment:
Should this be embedded on chain or injected by vit-ss?

Reply:
Injected by vit-ss at this point.